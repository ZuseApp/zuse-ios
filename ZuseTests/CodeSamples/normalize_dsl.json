{
  "suite": [
    { "set": ["#name", 1] },
    { "set": ["blah", "#value"] },
    { "set": ["#blah", "#value"] },
    { "set": ["this", "that"] },
    {
      "on_event": {
        "name": "#this",
        "parameters": [],
        "code": []
      }
    },
    {
      "on_event": {
        "name": "this",
        "parameters": [],
        "code": []
      }
    },
    {
      "trigger_event": {
        "name": "#this",
        "parameters": {}
      }
    },
    {
      "trigger_event": {
        "name": "this",
        "parameters": {}
      }
    },
    {
      "if": {
        "test": "#this",
        "true": [],
        "false": []
      }
    },
    {
      "if": {
        "test": true,
        "true": [],
        "false": []
      }
    },
    {
      "call": {
        "method": "foo",
        "parameters": ["foo", "#this", "that"]
      }
    },
    {
      "call": {
        "method": "foo",
        "parameters": ["foo", "this", "that"]
      }
    }
  ]
}
