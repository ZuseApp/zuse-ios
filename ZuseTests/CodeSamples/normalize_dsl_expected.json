{
  "suite": [
    { "set": ["this", "that"] },
    {
      "on_event": {
        "name": "this",
        "parameters": [],
        "code": []
      }
    },
    {
      "trigger_event": {
        "name": "this",
        "parameters": {}
      }
    },
    {
      "if": {
        "test": true,
        "true": [],
        "false": []
      }
    }
  ]
}
